libs/api/Verifier.cpp

```diff
@@ -513,7 +513,9 @@ ebpf_api_elf_enumerate_programs(
                     return 1;
                 }
                 auto& program = std::get<InstructionSeq>(programOrError);
-                cfg_t controlFlowGraph = prepare_cfg(program, raw_program.info, true);
+                prepare_cfg_options options = verifier_options.cfg_opts;
+                options.must_have_exit = true;
```

Comment:
The must_have_exit defaults to on, so why do we need to set it?

Reply:
Updated, thanks.